src/main/java/roomescape/domain/regex/BaseFormat.java

```diff
@@ -0,0 +1,7 @@
+package roomescape.domain.regex;
+
+public class BaseFormat {
+
+    public final static String EMAIL = "[a-zA-Z0-9+-\\_.]+@[a-zA-Z0-9-]+\\.[a-zA-Z0-9-.]+";
+    public final static String PASSWORD = "^(?=.*[a-zA-Z])(?=.*\\d)(?=.*[\\W_]).{8,}$";
```

Comment:
modifier 순서를 지켜주세요 😄 

```suggestion
    public static final String EMAIL = "[a-zA-Z0-9+-\\_.]+@[a-zA-Z0-9-]+\\.[a-zA-Z0-9-.]+";
    public static final String PASSWORD = "^(?=.*[a-zA-Z])(?=.*\\d)(?=.*[\\W_]).{8,}$";
```

참고 : https://google.github.io/styleguide/javaguide.html#s4.8.7-modifiers

Reply:
앗... 바로 수정하겠습니다!